backend/src/main/routes/AccountRoute.ts

```diff
@@ -140,20 +151,9 @@ export default class AccountRoute extends CommonRoutesConfig {
         }
       });
 
-    this.getApp()
-      .route(`/accounts/employee`)
-      .get(async (req: express.Request, res: express.Response, next: express.NextFunction) => {
-        try {
-          const regexEmployeeAccount = await this.employeeAccountService.getEmployeesByRegex(String(req.query.email));
-          res.status(StatusCodes.OK).send(regexEmployeeAccount);
-        } catch (err) {
-          next(err);
-        }
-      });
-
     this.getApp()
       .route(`/accounts/client`)
-      .all(checkJwt, checkRole(new Set([Roles.BUSINESS])))
+      .all(checkJwt)
```

Comment:
Why was the role restriction removed here?

Reply:
I believe this is because it was pinging auth0 too many time so we would get a 429 error. Alternative solution is to update the query to prevent the security issue I pointed out last time.

Reply:
yeah, auth0 wouldnt allow the search function to make multiple request in a short amount of time